Enrise/accessibility-checker-infrastructure
# SAC 17 - Add Travis CI to build the project and put up on GitHub pages

# What
- Added Travis CI to build the project and put up on GitHub pages

# How to test
- Check the Travis CI build runs
- Check the example sites are available via GitHub pages
